editions.go

```diff
@@ -95,6 +95,23 @@ func GetEdition(olid string) (b Book, err error) {
 	return
 }
 
+// GetEditionISBN returns a book from its isbnid
+func GetEditionISBN(isbnid string) (b Book, err error) {
+	s := fmt.Sprintf("https://openlibrary.org/isbn/%s.json", isbnid)
+	resp, err := http.Get(s)
+	if err != nil {
+		return b, err
```

Comment:
I suspect that `err` can be used to see the Openlibrary throws a 404 when you try to access a book with an isbn that is numerical

Reply:
I've made some testing, it turns out that you `err` will not be helpful. But you can use `resp.StatusCode` to check if it is 404 or not.